bin/passenger-recycler

```diff
@@ -41,31 +41,39 @@ def process_status?(pid)
 end
 
 require 'phusion_passenger'
+PhusionPassenger.locate_directories
 require 'phusion_passenger/platform_info'
 require 'phusion_passenger/platform_info/ruby'
 require 'phusion_passenger/admin_tools/memory_stats'
-PhusionPassenger.locate_directories
 stats = PhusionPassenger::AdminTools::MemoryStats.new
-unless stats.platform_provides_private_dirty_rss_information?
-  puts 'Please run as root or platform unsupported'
-  exit 1
-end
 killed = 0
 stats.passenger_processes.each do |p|
   pid = p.pid.to_i
-  debug "Checking #{pid} with RSS of #{p.private_dirty_rss}"
-  next unless p.private_dirty_rss > CONFIG[:MAX_PRIV_RSS_MEMORY]
   started = begin
               `ps -p#{pid} -o start=`.strip
             rescue StandardError => e
               verbose "Error: #{e.message} \nReturning '?'"
               '?'
             end
```

Comment:
Move this `ps` call back after `next unless` so it does not need to be call every loop.